bucket/keepass-plugin-keepassrpc.json

```diff
@@ -0,0 +1,16 @@
+{
+    "homepage": "https://www.kee.pm/",
+    "description": "Kee is a Firefox and Chrome add-on for linking browsers to KeePass, using the KeePassRPC KeePass plugin contained within this repository.",
+    "license": "GPL-2.0-or-later",
+    "version": "1.8.0",
+    "url": "https://github.com/kee-org/keepassrpc/releases/download/v1.8.0/KeePassRPC.plgx",
+    "hash": "8d9d5e390fc4a3b8d6d8f24dd26a712dc032c4ff49708c8ec32c95a2e27594b5",
+    "depends": "extras/keepass",
+    "checkver": {
+        "github": "https://github.com/kee-org/keepassrpc"
+    },
+    "autoupdate": {
+        "url": "https://github.com/kee-org/keepassrpc/releases/download/v$version/KeePassRPC.plgx"
+    },
+    "post_install": "Copy-Item \"$dir\\KeePassRPC.plgx\" \"$(appdir keepass)\\current\\Plugins\""
```

Comment:
I made this by example in [notepadplusplus-pm](https://github.com/lukesampson/scoop-extras/blob/c064d4d5d9d0773fdafb46ef831165d5bc00163e/bucket/notepadplusplus-pm.json). But I think it should be in `install` section, should I move it there?

Should I also make an uninstall section?

Reply:
Yes, the user will expect uninstall to remove the .plgx file from the Plugins folder.